app/Http/Controllers/AirlineController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Airline;
+use App\Models\City;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+use Illuminate\View\View;
+
+class AirlineController extends Controller
+{
+    public function index(): View
+    {
+        return view('airlines.show', [
+            'links' => Airline::paginate()->links(),
+            'cities' => City::all()
+        ]);
+    }
+
+    public function store(Request $request): JsonResponse
+    {
+        $attributes = $request->validate([
+            'name' => ['required', Rule::unique('airlines')],
+            'description' => ['nullable'],
+            'cities' => ['nullable']
+        ]);
+        $airline = Airline::create([
+            'name' => $attributes['name'],
+            'description' => $attributes['description']
+        ]);
+
+        if (isset($attributes['cities'])) {
+            $cities = City::whereIn('name', $attributes['cities'])->get();
+            $airline->cities()->attach($cities);
```

Comment:
```suggestion
            $airline->cities()->attach($attributes['cities']);
```
I'm not entirely sure if this works. Try it out and let me know.

Reply:
Nope, it doesn't works :(

Reply:
Oh, okay 😢